src/language-js/printer-estree.js

```diff
@@ -5337,24 +5337,73 @@ function isObjectType(n) {
 }
 
 // eg; `describe("some string", (done) => {})`
-function isTestCall(n) {
+function isTestCall(n, parent = null) {
```

Comment:
We do support node 4 so we cannot use `default` syntax here.

Reply:
@ikatyang fixed